generators/app/index.js

```diff
@@ -254,11 +295,16 @@ module.exports = class extends BaseGenerator {
                     this.jhipsterVersion = this.config.get('jhipsterVersion');
                 }
                 this.otherModules = this.config.get('otherModules') || [];
-                if (this.blueprint) {
-                    this.blueprintVersion = this.findBlueprintVersion(this.blueprint);
+                if (this.blueprints && this.blueprints.length > 0) {
+                    this.blueprints.forEach(blueprint => {
+                        blueprint.version = this.findBlueprintVersion(blueprint.name) || 'latest';
+                    });
+
                     // Remove potential previous value to avoid duplicates
-                    this.otherModules = this.otherModules.filter(module => module.name !== this.blueprint);
-                    this.otherModules.push({ name: this.blueprint, version: this.blueprintVersion || 'latest' });
+                    this.otherModules = this.otherModules.filter(module => {
+                        return this.blueprints.findIndex(bp => bp.name === module.name) === -1;
```

Comment:
why is return needed? it can use shorter syntax as earlier

Reply:
Fixed.